dimod/binary/binary_quadratic_model.py

```diff
@@ -2454,13 +2462,34 @@ def BinaryArray(labels: Union[int, Iterable[Variable]],
     single binary variable.
 
     Args:
-        labels: Either an iterable of variable labels or a number. If a number
-            labels are generated using :class:`uuid.UUID`.
+        labels: Either an iterable of variable labels or the number of required
+            models. If a number, labels are generated using :class:`uuid.UUID`.
         dtype: Data type for the returned binary quadratic models.
 
     Returns:
-        Array of binary quadratic models, each with a single binary variable.
+        Array of :class:`.BinaryQuadraticModel`\ s, each representing a single
+        binary variable.
+
+    Examples:
+        This example generates the QUBO, :math:`{x}^{T} {Q}{x}`, for a Boolean
```

Comment:
I like this example, I think `Binaries` and `Binary` should get a similar treatment.

Reply:
Thanks, that's the plan for the next push.

Reply:
Done